index.js

```diff
@@ -68,8 +68,12 @@ cli.on('require', function(name) {
   log.info('Requiring external module', ansi.magenta(name));
 });
 
-cli.on('requireFail', function(name) {
-  log.warn(ansi.yellow('Failed to load external module'), ansi.magenta(name));
+cli.on('requireFail', function(name, error) {
+  var message = name + (error ? '\n    [' + error + ']' : '');
+  log.warn(
+    ansi.yellow('Failed to load external module'),
+    ansi.magenta(message)
```

Comment:
@smialy I'm sorry but I want you to change more.

1. Change color of error message to yellow. Magenta is for module name and this warning message should be yellow because we changed this color to yellow by user's request.
2. Do `log.warn()` each line separately so as to put timestamp at the beginning of each line, like other log outputs.